lib/matplotlib/rcsetup.py

```diff
@@ -310,6 +310,16 @@ def validate_backend(s):
     _deprecated_since="3.3")
 
 
+def _validate_toolbar(s):
+    s = ValidateInStrings(
+        'toolbar', ['None', 'toolbar2', 'toolmanager'], ignorecase=True)(s)
+    if s == 'toolmanager':
+        cbook._warn_external(
```

Comment:
Should be 
```suggestion
        _api.warn_external(
```

Reply:
Yes, I already had realized the cbook / _api issue, but I also had to adjust the tests
(test_backend_bases.py::test_toolbar_zoompan, please revise if the changes are ok)

Now, pytest passes, but an Azure Pipeline fails. I don't know what I can do about that and if it is a serious issue. Any chance you have a look at it?